packages/admin/test/e2e/cli.feature.ts

```diff
@@ -110,14 +110,25 @@ describe('CLI', () => {
     expect(sdkCliDiff).toEqual(uncoveredFunctions);
   });
 
+  describe('derive-airnode-xpub', () => {
+    it('derives airnode xpub', () => {
+      const airnodeMnemonic = airnodeWallet.mnemonic.phrase;
+
+      // Derive the xpub programatically
+      const airnodeXpub = admin.deriveAirnodeXpub(airnodeMnemonic);
+
+      // Derive the wallet using CLI
+      const out = execCommand('derive-airnode-xpub', ['--airnode-mnemonic', airnodeMnemonic]);
```

Comment:
I wonder how this is passing 🤔 

```suggestion
      const out = execCommand('derive-airnode-xpub', ['--mnemonic', airnodeMnemonic]);
```